### DISCUSSION TITLE
windows ink pressure sensitivity doesnt work

### DISCUSSION THREAD

**Discussion by SempaiYeet** [OpenTabletDriver/OpenTabletDriver#1247] [Q&A] [answered]
When using the windows ink absolute mode, everything works but when I press down there's no pressure sensitivity, it just registers as if I pressed down hard.

**Comment by jamesbt365** [comment, marked as answer]
what program are you using, most programs you will need to change what api they use.

In Krita you need to go settings > configure krita > tablet and change it to windows ink labeld either as tablet pc or windows 8.

If it works here but not in photoshop then you will need to reinstall photoshop and this time install it with windows ink enabled due to photoshop not allowing you to change it whenever.

Paint Tool Sai doesn't support windows ink and therefore you cannot use it with pressure.

Clip studio paint go to File > preferences > tablet
